src/data/pets.json

```diff
@@ -16,6 +16,10 @@
       "name": "Dobby",
       "image": "/images/pets/dobby.jpg"
     },
+    {
+      "name": "Eevee",
```

Comment:
I do think these are intended to be in alphabetical order, so Eevee would go under Echo. Could you move Catnip above DD as well?